doc/Manuals/ig-cs_x-road_6_central_server_installation_guide.md

```diff
@@ -23,85 +25,84 @@ Doc. ID: IG-CS
   * [1.3 References](#13-references)
-- [2. Installation](#2-installation)
+* [2. Installation](#2-installation)
   * [2.1 Prerequisites to Installation](#21-prerequisites-to-installation)
   * [2.2 Reference Data](#22-reference-data)
   * [2.3 Requirements to the Central Server](#23-requirements-to-the-central-server)
   * [2.4 Preparing OS](#24-preparing-os)
   * [2.5 Installation](#25-installation)
   * [2.6 Installing the Support for Hardware Tokens](#26-installing-the-support-for-hardware-tokens)
   * [2.7 Installing the Support for Environmental Monitoring](#27-installing-the-support-for-environmental-monitoring)
-- [3 Initial Configuration](#3-initial-configuration)
+* [3 Initial Configuration](#3-initial-configuration)
   * [3.1 Reference Data](#31-reference-data)
   * [3.2 Initializing the Central Server](#32-initializing-the-central-server)
   * [3.3 Configuring the Central Server and the Management Services' Security Server](#33-configuring-the-central-server-and-the-management-services-security-server)
-- [4 Additional configuration](#4-additional-configuration)
+* [4 Additional configuration](#4-additional-configuration)
   * [4.1 Adding support for V1 global configuration](#41-adding-support-for-v1-global-configuration)
-- [5 Installation Error Handling](#5-installation-error-handling)
+* [5 Installation Error Handling](#5-installation-error-handling)
   * [5.1 Cannot Set LC_ALL to Default Locale](#51-cannot-set-lc_all-to-default-locale)
   * [5.2 PostgreSQL Is Not UTF8 Compatible](#52-postgresql-is-not-utf8-compatible)
   * [5.3 Could Not Create Default Cluster](#53-could-not-create-default-cluster)
   * [5.4 Is Postgres Running on Port 5432?](#54-is-postgres-running-on-port-5432)
 
+<!-- vim-markdown-toc -->
 <!-- tocstop -->
 
-
 ## License
 
 This document is licensed under the Creative Commons Attribution-ShareAlike 3.0 Unported License. To view a copy of this license, visit http://creativecommons.org/licenses/by-sa/3.0/.
 
 
-# 1. Introduction
+## 1. Introduction
 
-## 1.1 Target Audience
+### 1.1 Target Audience
 
 The intended audience of this installation guide are the X-Road central server administrators responsible for installing and configuring the X-Road central server software.
 The document is intended for readers with a good knowledge of Linux server management, computer networks, and the X-Road functioning principles.
 
-
-## 1.2 Terms and abbreviations
+### 1.2 Terms and abbreviations
 
 See X-Road terms and abbreviations documentation \[[TA-TERMS](#Ref_TERMS)\].
 
-## 1.3 References
+### 1.3 References
 
 1. <a id="Ref_UG-CS" class="anchor"></a>\[UG-CS\] Cybernetica AS. X-Road 6. Central Server User Guide. Document ID: [UG-CS](ug-cs_x-road_6_central_server_user_guide.md) 
 2. <a id="Ref_IG-SS" class="anchor"></a>\[IG-SS\] Cybernetica AS. X-Road 6. Security Server Installation Guide. Document ID: [IG-SS](ig-ss_x-road_v6_security_server_installation_guide.md)
```

Comment:
Link should be updated: 
ig-ss_x-road_v6_security_server_installation_guide.md
=>
ig-ss_x-road_v6_security_server_installation_guide_for_ubuntu.md

This applies to all the `md` documents too.

Reply:
Should we actually keep the old name (ig-ss_x-road_v6_security_server_installation_guide.md) for the Ubuntu installation guide? There can be external links which will break if we change the name.

Reply:
Yes, let's keep the old name. In that way we don't break any external links - there are probably many of them.